src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -809,6 +1079,94 @@ def get_rom_directory(self, rom_name):
 
         return os.path.join(self.tbrom_directory_path, rom_name)
 
+    def get_rom_nslist(self, rom_name):
+        """
+        Get the list of named selections associated to the TBROM named rom_name
```

Comment:
```suggestion
        Get the list of named selections associated with a TBROM.
```

Reply:
yes in principle those are the same but we should stick to TBROM only in this context